CHANGELOG.md

```diff
@@ -1,6 +1,69 @@
 # Changelog
 Notable changes to the framework should be documented here
 
+3.12.3:
+	- - fixup FR push_over_rsync for FR i2 (ENT-4587)
+	- Added AIX support to body perms system_owned (ENT-4773)
+	- Added ability to avoid limiting robot agents (CFE-3161)
+	- Added and transitioned to using master_software_updates shortcut
+	  (ENT-4953)
+	- Added continual checking for policy_server state (CFE-3073)
+	- Added documentation how to enable systemd unit management and disable agents on all hosts
+	  (CFE-3416)
+	- Added package_module for snap (CFE-2811)
+	- Added scripts and templates for Federated Reporting (ENT-4473)
+	- Added support for 'awk' filters in the FR dump-import process
+	  Ticket: (ENT-4839)
```

Comment:
Style, this one says Ticket: (BLAH) but none of the others do, that's kind of strange.